tests/Unit/PointwiseFunctions/AnalyticData/GrMhd/Test_CylindricalBlastWave.cpp

```diff
@@ -0,0 +1,230 @@
+       "cylindrical_blast_wave_pressure",
+       "cylindrical_blast_wave_lorentz_factor",
+       "cylindrical_blast_wave_specific_enthalpy",
+       "cylindrical_blast_wave_magnetic_field",
+       "cylindrical_blast_wave_divergence_cleaning_field"},
+      {{{-1.1, 1.1}}}, member_variables, used_for_size);
+}
+
+// Check points on and near boundaries. Check that density = inner_density
+// at r = inner radius and at r = inner_radius - epsilon, and check that
+// density is approximately inner_density at r = inner_radius + epsilon.
+// Then do the analogous check for points on and near outer_radius.
+void test_density_on_and_near_boundaries() {
+  const double inner_radius = 0.8;
+  const double outer_radius = 1.0;
+  const double inner_density = 1.0e-2;
+  const double outer_density = 1.0e-4;
+  const double inner_pressure = 1.0;
+  const double outer_pressure = 5.0e-4;
+  const std::array<double, 3> magnetic_field{{0.1, 0.0, 0.0}};
+  const double adiabatic_index = 1.3333333333333333333;
+
+  CylindricalBlastWaveProxy cylindrical_blast_wave(
+      inner_radius, outer_radius, inner_density, outer_density, inner_pressure,
+      outer_pressure, magnetic_field, adiabatic_index);
+
+  const double epsilon = 1.e-10;
+  Approx approx = Approx::custom().epsilon(epsilon * 100.0);
+  auto x =
+      make_with_value<tnsr::I<double, 3, Frame::Inertial>>(inner_radius, 0.0);
+
+  get<0>(x) = inner_radius;
+  CHECK(inner_density == get(get<hydro::Tags::RestMassDensity<double>>(
+                             cylindrical_blast_wave.grmhd_variables(x))));
+  get<0>(x) = inner_radius - epsilon;
+  CHECK(inner_density == get(get<hydro::Tags::RestMassDensity<double>>(
+                             cylindrical_blast_wave.grmhd_variables(x))));
+  get<0>(x) = inner_radius + epsilon;
+  CHECK_ITERABLE_CUSTOM_APPROX(inner_density,
+                               get(get<hydro::Tags::RestMassDensity<double>>(
+                                   cylindrical_blast_wave.grmhd_variables(x))),
+                               approx);
+
+  get<0>(x) = outer_radius;
+  CHECK(outer_density == get(get<hydro::Tags::RestMassDensity<double>>(
+                             cylindrical_blast_wave.grmhd_variables(x))));
+  get<0>(x) = outer_radius + epsilon;
+  CHECK(outer_density == get(get<hydro::Tags::RestMassDensity<double>>(
+                             cylindrical_blast_wave.grmhd_variables(x))));
+  get<0>(x) = outer_radius - epsilon;
+  CHECK_ITERABLE_CUSTOM_APPROX(outer_density,
+                               get(get<hydro::Tags::RestMassDensity<double>>(
+                                   cylindrical_blast_wave.grmhd_variables(x))),
+                               approx);
+}
+}  // namespace
+
+SPECTRE_TEST_CASE(
+    "Unit.PointwiseFunctions.AnalyticData.GrMhd.CylindricalBlastWave",
+    "[Unit][PointwiseFunctions]") {
+  pypp::SetupLocalPythonEnvironment local_python_env{
+      "PointwiseFunctions/AnalyticData/GrMhd"};
```

Comment:
@wthrowe Thanks...this is the line causing the tests to fail. I was requested to write the full path out on line 203, and when I did, the test started failing. I pushed anyway, to make sure it wasn't something screwed up on my laptop. The error I get is "ImportError: No module named AnalyticData.GrMhd.TestFunctions". I hope when I rebase on develop that this error goes away, but do you know why this is happening?